Uyanda.Coffee.Application/Features/BeverageManagement/Persistence/IBeverageAccessor.cs

```diff
@@ -10,5 +10,6 @@ public interface IBeverageAccessor
         Task<BeverageSizeCostModel[]> AddBeverageCostAsync(IEnumerable<BeverageSizeCostModel> prices);
         Task<BeverageSizeCostModel[]> GetBeverageCostAsync();
         Task<InvoiceModel> PurchaseAsync(IEnumerable<LineItemModel> lineItems);
+        Task<BeverageSizeCostModel[]> AddBeverageSizeCostAsync(BeverageSizeCostModel price);
```

Comment:
Rather name these chain of methods _UpsertBeverageSizeCostAsync_. As the methods not only add (insert), but also update. That is called **Upsert**